### DISCUSSION TITLE
Sqlite example brakes after some time

### DISCUSSION THREAD

**Discussion by JustLinuxUser** [maxcountryman/axum-login#233] [Q&A]
Hello, I have been playing with the sqlite auth example, and after some time I get an error: `no such table users`
Here is what I changed: 
[changes.diff.txt](https://github.com/user-attachments/files/16012733/changes.diff.txt)
Here is the full log:
[log.txt](https://github.com/user-attachments/files/16012726/log.txt)

But from what I remember I could reproduce the issue without any changes

**Comment by maxcountryman** [comment]
This is related to `sqlx` and specifically how the in-memory SQLite database is handled. You can try using a file descriptor instead, which will ensure that the memory representation is not lost or reset.

If you're continuing to have issues with `sqlx` and SQLite, you may want to seek advice from the `sqlx` support channels.

**Reply by JustLinuxUser** [reply]
but I didn't reset anything, anyway, thanks for the response
